packages/components/src/components/combobox/style.scss

```diff
@@ -0,0 +1,42 @@
+@import '../@shared/mixins';
+@import '../input-line';
+
+@layer kol-component {
+	:host {
+		font-size: rem(16);
+	}
+
+	.combobox {
+		position: relative;
+		&.disabled,
+		&.disabled * {
+			cursor: not-allowed !important;
+		}
+		&.disabled {
+			opacity: 0.5;
+			outline: none;
+		}
+		&__group {
+			display: inline-flex;
+			align-items: center;
+			cursor: pointer;
+		}
+
+		&__input {
+			flex-grow: 1;
+		}
+
+		&__listbox {
+			display: block;
+			position: absolute;
+			list-style-type: none;
+			margin: 0;
+			padding: 0;
+			overflow-y: auto;
+			overflow-x: hidden;
+			z-index: 2;
+			background-color: white;
+			max-height: 15.625rem;
```

Comment:
Hier müssen wir die `rem()`-Funktion verwenden (Siehe L6), sonst skaliert die Komponente ggf. falsch.

Reply:
Die rem-Werte müsstest du dir noch einmal anschauen: Die rem-Funktion rechnet praktisch von px nach rem um. Du musst die Zahlen also entsprechend umrechnen, bevor du sie in die `rem`-Funktion gibst.
Aus `15.625rem` wird z.B. `rem(250)`